src/pages/myAppointments.tsx

```diff
@@ -54,18 +54,24 @@ export default function Home() {
         <Box sx={{ marginTop: "3vh" }}>
           <Typography variant="h1">My Appointment</Typography>
         </Box>
-        <Grid
-          container
-          justifyContent="space-between"
-          rowSpacing={6}
-          style={{ width: "80vw", marginTop: -6 }}
-        >
-          {appointments.map((appointment, index) => (
-            <Grid item key={index} xs={5.75}>
-              <AppointmentCard appointment={appointment} prefix={PAGE_PATHS.APPOINTMENT} />
-            </Grid>
-          ))}
-        </Grid>
+        {appointments.length === 0 ? (
```

Comment:
Not sure is it backend error or not. This didn't work for me so I try console log `appointments.length` and it show `400 Bad Request`. Could you check whether this happen to you or not? If not may be it might be my browser security problem.